course05/homework01/skeleton_twyleg/string.h

```diff
@@ -0,0 +1,41 @@
+// Some copyright 2020
+
+#pragma once
+
+#include <string>
+#include <ostream>
+#include <array>
+#include <cstring>
+
+namespace cppcourse {
+
+class String
+{
+public:
+	String();
+    String(std::string);
+
+    bool Empty() const { return (mSize == 0); };
+    std::size_t Size() const { return mSize; };
+    const char* c_str() const { return mData.data(); };
+    String& append(const String&);
+
+
+    friend std::ostream& operator<<(std::ostream& stream, String str);
+    friend bool operator==(const String& lhs, const String& rhs);
+    friend bool operator!=(const String& lhs, const String& rhs);
```

Comment:
Why do they need to be friend?

Reply:
- [ ] Make `==` and `!=` member functions